src/org/ensembl/healthcheck/testcase/compara/CheckTopLevelDnaFrag.java

```diff
@@ -72,6 +72,15 @@ public boolean checkTopLevelDnaFrag(DatabaseRegistryEntry comparaDbre) {
 		boolean result = true;
 		Connection comparaCon = comparaDbre.getConnection();
 
+		// LRGs are only required by vertebrates division
+		String division = DBUtils.getRowColumnValue(comparaCon, "SELECT meta_value FROM meta WHERE meta_key = 'division'");
```

Comment:
This is the same query as in https://github.com/Ensembl/ensj-healthcheck/blob/c1282951989b55a55ad3f7a1c0cee82896e5f7e0/src/org/ensembl/healthcheck/testcase/compara/CheckGenomicAlignCoverage.java#L73-L74
It should be moved to the parent class AbstractComparaTestCase. But then you'll notice that the latter already has `getComparaDivisionName`, which was written before the meta key was introduced. https://github.com/Ensembl/ensj-healthcheck/blob/c1282951989b55a55ad3f7a1c0cee82896e5f7e0/src/org/ensembl/healthcheck/testcase/compara/AbstractComparaTestCase.java#L232-L254
As I'm not sure all the EG databases will have the meta-key correctly set, I would suggest to make `getComparaDivisionName` read the meta-key first, and fall back to pattern matching on the database name. BTW, it should now return "vertebrates", not "ensembl".